File: backend/workspaces/sessions/test-python-1/task.py
class Task:
    """Represents a single task."""
    def __init__(self, id: int, title: str, priority: str):
        self.id = id
        self.title = title
        self.priority = priority
        self.done = False

    def to_dict(self):
        return {
            "id": self.id,
            "title": self.title,
            "priority": self.priority,
            "done": self.done
        }

File: backend/workspaces/sessions/test-java-1/Task.java
public class Task {
    public int id;
    public String title;
    public String priority;
    public boolean done;

    public Task(int id, String title, String priority) {
        this.id = id;
        this.title = title;
        this.priority = priority;
        this.done = false;
    }
}
